_posts/news/2022-02-13-announcing-scalajs-1.9.0.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: Announcing Scala.js 1.9.0
+category: news
+tags: [releases]
+permalink: /news/2022/02/13/announcing-scalajs-1.9.0/
+---
+
+
+We are excited to announce the release of Scala.js 1.9.0!
+
+Starting with this release, Scala.js will use its strict-floats mode by default (what previously required `withStrictFloats(true)`).
+`Float` values are now always guaranteed to fit in 32-bit floating point data, and all `Float` operations strictly follow IEEE-754 float32 semantics.
+
+This release also brings support for `java.util.BitSet`, and fixes some bugs.
+It also updates the version of the Scala standard library to 2.13.8 for 2.13.x versions.
+
+Read on for more details.
+
+<!--more-->
+
+## Getting started
+
+If you are new to Scala.js, head over to [the tutorial]({{ BASE_PATH }}/tutorial/).
+
+If you need help with anything related to Scala.js, you may find our community [in `#scala-js` on Discord](https://discord.com/invite/scala) and [on Stack Overflow](https://stackoverflow.com/questions/tagged/scala.js).
+
+Bug reports can be filed [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Release notes
+
+If upgrading from Scala.js 0.6.x, make sure to read [the release notes of Scala.js 1.0.0]({{ BASE_PATH }}/news/2020/02/25/announcing-scalajs-1.0.0/) first, as they contain a host of important information, including breaking changes.
+
+This is a **minor** release:
+
+* It is backward binary compatible with all earlier versions in the 1.x series: libraries compiled with 1.0.x through 1.8.x can be used with 1.9.0 without change.
+* It is *not* forward binary compatible with 1.8.x: libraries compiled with 1.9.0 cannot be used with 1.8.x or earlier.
+* It is *not* entirely backward source compatible: it is not guaranteed that a codebase will compile *as is* when upgrading from 1.8.x (in particular in the presence of `-Xfatal-warnings`).
+
+As a reminder, libraries compiled with 0.6.x cannot be used with Scala.js 1.x; they must be republished with 1.x first.
+
+## Strict floats by default
```

Comment:
I think this should mention somewhere that strict floats is a linking time decision (and therefore applies to the whole program).